zipline/algorithm.py

```diff
@@ -418,6 +453,32 @@ def before_trading_start(self, data):
 
         self._in_before_trading_start = False
 
+    def process_capital_changes(self, dt):
```

Comment:
Nothing about this depends on anything in the algorithm.  Can we just push this method down into `PerformanceTracker` and have `AlgorithmSimulator` call it directly?